.travis.yml

```diff
@@ -7,35 +7,35 @@ matrix:
     # Note: The ordering of the environments is optimized so that the complete matrix runs as quickly as possible,
     # while also providing the first results as early as possible.
     # At the moment, build times for each environment are roughly as follows:
-    # xcode6.4: 13 minutes
-    # xcode7.3: 5 minutes
-    # xcode8.3: 3 minutes
-    # xcode9.2: 2 minutes
+    # xcode7.3: 16 minutes
+    # xcode8.3: 7 minutes
+    # xcode9.2: 7 minutes
+    # xcode10.1: 6 minutes
+    # https://docs.travis-ci.com/user/reference/osx/#macos-version
     # This data is based on the following build, which should be similar to most builds on the repo:
     # https://travis-ci.org/pybee/rubicon-objc/builds/348992202
```

Comment:
Could you change this build URL to the one from where you got the new run times?